plugins/w3c_logs.yaml

```diff
@@ -35,6 +35,47 @@ parameters:
     description: Max number of W3C files that will be open during a polling cycle
     type: int
     default: 512
+  - name: timestamp_layout
+    description: Optional timestamp layout which will parse a timestamp field
+    type: string
+  - name: timestamp_parse_from
+    description: Field to parse the timestamp from, required if 'timestamp_layout' is set
+    type: string
+  - name: timestamp_layout_type
+    description: Optional timestamp layout type for parsing the timestamp, suggested if 'timestamp_layout' is set
+    type: string
+    supported:
+      - strptime
+      - gotime
+      - epoch
+    default: strptime
+  - name: timestamp_location
+    description: The geographic location (timezone) to use when parsing a timestamp that does not include a timezone. The available locations depend on the local IANA Time Zone database.
+    type: string
+    default: UTC
+  - name: parse_from
+    description: Where to parse the data from
+    type: string
+    default: body
+  - name: parse_to
+    description: Where the data will parse to
+    type: string
+    supported:
+      - attributes
+      - body
+    default: body
+  - name: lazy_quotes
+    description: If true, a quote may appear in an unquoted field and a non-doubled quote may appear in a quoted field. Cannot be true if 'ignore_quotes' is true.
+    type: bool
+    default: false
+  - name: ignore_quotes
+    description: If true, all quotes are ignored, and fields are simply split on the delimiter. Cannot be true if 'lazy_quotes' is true.
+    type: bool
+    default: false
```

Comment:
W3C logs are ALWAYS whitespace separated. The spec for W3C doesn't mention quoting, they are simply "whitespace separated", so we should ALWAYS ignore quotes if they are valid W3C logs. Have we actually run into a situation where we would want to respect quotes for W3C logs in the field?

(if we really need this functionality, I suggest having `ignore_quotes` true by default, so we follow the spec by default)

Reply:
Per your reference to the spec, I've removed the quotes options.